plugins/aea-cli-benchmark/aea_cli_benchmark/case_acn_startup/utils.py

```diff
@@ -0,0 +1,245 @@
+#!/usr/bin/env python3
```

Comment:
?? This whole file contains duplicated code from what we have in conftest.py already!